src/crimson/os/seastore/lba_manager/btree/btree_lba_manager.cc

```diff
@@ -734,52 +733,39 @@ BtreeLBAManager::_decref_intermediate(
   extent_len_t len)
 {
   auto c = get_context(t);
-  return with_btree<LBABtree>(
+  return with_btree_ret<LBABtree, update_mapping_ret_bare_t>(
```

Comment:
I'm curious, why change `with_btree` to `with_btree_ret`? Looks both versions need to return values.